README.md

```diff
@@ -69,12 +69,12 @@ msys2 -mingw32 -c 'echo $(gimptool-2.0 -n --build fourier.c) -lfftw3 -O3 | sh'
 This is for 32bits version ; replace i686 by x84_64 and -mingw32 by -mingw64 if you want 64bits. 
 Replace also 2.10.24 by your GIMP version (or leave empty for latest version)
 
-Also, the windows binaries are build through GitHub Actions. So you may also fork this repository and build the plugin on your own.
+Also, the windows binaries are built through GitHub Actions, so you may also fork this repository and build the plugin on your own.
 
 
 ## Maintainers:
 
-To create a distributable fourier-{version}.tar.gz file, you  will need to do these steps:
+To create a distributable gimp-fourier-plugin-{version}.tar.gz file, you  will need to do these steps:
```

Comment:
Will be a gimp-plugin-fourier-[version}.tar.gz if I am not mistaken?

I am a bit shameful about naming: I did not care too much when creating the initial zip file and then the github repo, hence the name variety, but I think renaming this repo is the last thing to do. To sum up to see if I am right:
- repo name: plugin-gimp-fourier
- autotools tar.gz: gimp-plugin-fourier
- deb package: gimp-plugin-fourier 
- rpm package: gimp-fourier-plugin

Right?

Reply:
yeah - learn from trial and error. It is a plate of sphagetti.

I don't think this project is being followed yet.
Maybe start another repo folder, and get your main home page to point to the new repo, and try to keep things more-or less aligned. This is probably a good starting point on seeing what everyone agrees with.
https://repology.org/project/gimp:fourier/versions

Would be a lot of effort on your side, but in the end, it's cleaned-up.
A fresh repo would also not have all the extra commits this repo currently has at the start. I've made my share of errors too, and for example added Stopped as the last commit here: https://github.com/JoesCat/xinv3d